---
string Returncode_t
int32[] line
#int32[] curve


